code/modules/cm_tech/droppod/droppod.dm

```diff
@@ -157,6 +158,10 @@
 	for(var/obj/structure/structure in loc)
 		structure.update_health(-land_damage)
 
+		// Deal damage exclusively to barricades/tables on landing, used for sentry post
+		if(istype(structure, /obj/structure/barricade) || istype(structure, /obj/structure/surface/table))
```

Comment:
this could probably be an `is_type_in_list` with the list being a static one on the drop pod

Reply:
did is_type_in_list and created a new variable in droppod that dictates what objects they apply on, made it empty so that child objects will have to manually call what those are